### ISSUE TITLE
VA Telephone: aria-hidden=true in not clickable version makes this inaccessible for screen readers

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/vets-design-system-documentation#3747]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When you navigate the `<va-telephone>`, If the property is set `not-clickable="true"`, the text isn't reachable by a screen reader.

I verified this testing the [Notice of Disagreement form](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/contact-information). You can also see it in the [Design System Storybook](https://design.va.gov/storybook/?path=/docs/components-va-telephone--not-clickable).

## What I expected to happen

This is plain text, so it should be reachable via a screen reader. Currently, it's marked up like this in the web component:

```html
<span aria-hidden="true">877-345-9876</span>
```

That ARIA attribute isn't needed, and seems like a mistake.

## Reproducing

- CSS Library version: N/A
- Device: Safari with VoicerOver on desktop and iOS; Firefox with NVDA on Windows.
- Browser: Noted above.

Steps to reproduce:

1. Visit the [Design System Storybook](https://design.va.gov/storybook/?path=/docs/components-va-telephone--not-clickable) with Safari and VoiceOver.
2. Turn on VoiceOver.
3. Try to navigate to the phone number text.
4. You won't be able to hear the phone number announced.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

### Details

I would flag this as a `a11y-defect-1` since content can't be reached. It should be fixed quickly.

Here's a video of me testing this.

https://github.com/user-attachments/assets/ac6daf1f-c667-4059-9ade-86daf1995471

**Comment by xirclebox** [comment]
- https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-telephone/va-telephone.tsx

Ideally, to allow for this text to be exposed to all users, the more robust solution would be to simplify the markup to this.

```
<span>
  {formattedNumber}
</span>
```

**Comment by dfong-adh** [comment]
PR waiting for design system team to review.

**Comment by Mottie** [comment]
Fix merged in the content library. It will be available in vets-website after the next library update.

**Comment by davidakennedy** [comment]
@Mottie When will it be updated? Happy to test, if needed.

**Comment by Mottie** [comment]
@davidakennedy it looks like this is now available for validation.

**Comment by davidakennedy** [comment]
@Mottie @caw310 I retested the changes in our Notice of Disagreement form, and the phone numbers read out as expected. This ticket can be closed.

**Comment by caw310** [comment]
Closing per @davidakennedy comment.
